examples/bankaccount/README.md

```diff
@@ -1,4 +1,294 @@
 # Concordance Banking Example
+
 This example illustrates all of the core event sourcing building blocks applied to a simplified banking model. In this sample, we have modeled a bank account aggregate, projector, process manager, and an artificial gateway that can be used for testing simulated inter-bank transfers (processes/sagas).
 
-⚠️ _**Under Construction**_
\ No newline at end of file
+## Step by step guide
+
+To run these examples, use the following steps.
+
+<details>
+
+<summary>
+
+### 0. 📦 Download and set up `wash` and other dependencies
+
+</summary>
+
+To run [wasmCloud][wasmcloud] locally, you'll need to download and install `wash`:
+
+If you have the [rust toolchain][rust-toolchain] installed, you can use `cargo` to install wash:
+
+```console
+cargo install wash-cli
+```
+
+On a Ubuntu based system this you can use `apt`
+
+```console
+sudo apt install wash
+```
+
+For other options on how to install `wash`, see [the `wasmcloud/wash`][wash-docs-install].
+
+Along with `wash`, ensure you have access to the following:
+
+- [`nats` client](https://docs.nats.io/running-a-nats-service/clients) for communicating with NATS
+- [`jq`](https://stedolan.github.io/jq/) for displaying JSON
```

Comment:
I think we need to mention having redis installed/running

Reply:
- [ ] mention `redis-server`